data/eks_dockerimage-replication/versions/default.yaml

```diff
@@ -1,17 +1,4 @@
 charts:
-  nvidia_device_plugin:
```

Comment:
Is this safe to remove?

Reply:
It should be there...it is in a different location (at the bottom)